README.md

```diff
@@ -35,31 +35,36 @@ date, please check the [Gerrit VDSM project](https://gerrit.ovirt.org/q/project:
 Add oVirt repositories to your repositories list.
 
 For CentOS Stream 8 use:
-
-    sudo dnf copr enable -y ovirt/ovirt-master-snapshot centos-stream-8
-    sudo dnf install -y ovirt-release-master
+```shell
+$ sudo dnf copr enable -y ovirt/ovirt-master-snapshot centos-stream-8
+$ sudo dnf install -y ovirt-release-master
+```
```

Comment:
May be this looks better in pycharm but this format is harder to read and edit. Please keep the generic indented block as is.